samples/posix/uname/README.rst

```diff
@@ -0,0 +1,56 @@
+.. _posix-uname-sample:
+
+POSIX uname()
+########################################################
+
+Overview
+********
+
+In this sample application, the POSIX uname() function
```

Comment:
adding an hyperlink to uname()'s doc would be welcome
https://pubs.opengroup.org/onlinepubs/9699919799/functions/uname.html

Reply:
@kartben - still on the backburner, but I think we will eventually add doxygen for all posix api calls, with links to the upstream specs. It's just such a big endeavour, that I would probably want to do it in an automated way.

Reply:
@cfriedt you're suggesting writing original docs, right? Because I don't think the Open Group license would allow us to reuse/redistribute their documentation?